### PR TITLE
WT-6762 Use stats instead of files to check consumption.

### PR DESCRIPTION
**Pull request by sueloverso**
@ddanderson I changed this test to use statistics instead of file names to ensure consumption since files can be returned in any order and the sets may not be disjoint. I also fixed a strange indentation I found in there.

### CODE REVIEW

File: test/suite/test_bug019.py [hunk 1]
```diff
@@ -72,10 +78,10 @@ def populate(self, nentries):
...
         for i in range(1,90):
-                f = fnmatch.filter(os.listdir('.'), "*Prep*")
-                if f:
-                        return f
-                time.sleep(1.0)
+            f = fnmatch.filter(os.listdir('.'), "*Prep*")
+            if f:
+                return
+            time.sleep(1.0)
         self.assertFalse(not f)
```
Comment (ddanderson) [inline test/suite/test_bug019.py:85]: This double negative feels hard to parse.  I would use something like `self.fail('prep file not created')`

File: test/suite/test_bug019.py [hunk 2]
```diff
@@ -97,23 +103,22 @@ def test_bug019(self):
...
-            # pre-allocated files can persist longer than newer files when newer
-            # files are returned first. Confirm files are being consumed.
-            if set(older) < set(newer):
+            # Make sure we're consuming pre-allocated files.
+            if used >= newused:
                 self.pr("FAILURE on Iteration " + str(i))
-                self.pr("FAILURE: Older")
-                self.pr(str(older))
-                self.pr("FAILURE: Newer")
-                self.pr(str(newer))
```
Comment (ddanderson) [inline test/suite/test_bug019.py:113]: Probably easier to read as `self.pr("FAILURE: Older " + str(older) + " Newer " + str(newer))`

**Review by ddanderson — APPROVED** [review]
LGTM - some suggestions for readability

File: test/suite/test_bug019.py (hunk 2 above)
Comment (sueloverso) [reply test/suite/test_bug019.py:113]: Those lines are removed and replaced with others.
